src/extension/gogoduck/doc/wps-acorn.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I basically wanted to couple some sane parameters together, something that's easy to iterate on. So for ACORN if you use those parameters, you can a pretty quick response.